data/foia_source.mk

```diff
@@ -0,0 +1,4 @@
+data/output/foia-source-lookup.csv : data/raw/foia-source-lookup.csv data/raw/foia-source-lookup.full.csv
+	csvstack $^ | \
+	csvsql --query 'SELECT MIN(Employer), ID FROM STDIN GROUP BY ID' | \
```

Comment:
this is a little concerning. are there records with the same ID but different names and how sure are we that that is safe.

Reply:
thank you for this catch. i added a query to spit out instances where employer differed in more than casing for one id. many of the instances look like the same thing, however [there are some that are totally different](https://gist.github.com/hancush/a27174053859410ff71b99d95726bf06#file-duplicate-audit-csv-L18-L25). i think we need more information about this lookup (and how they assign the ids on their end) before we push this broader change forward. can we chat about an interim step to get the ad data out the door?

Reply:
sure. Call me at 1:45?

Reply:
will do, thanks!

Reply:
verdict: i am going to undo lookup-related changes and add ibhe to the original lookup file, which will resolve our immediate problem. for the future, i’ll open an issue to acquire a complete, canonical lookup table from bga.

Reply:
opened #325.